requirements.in

```diff
@@ -15,8 +15,8 @@ django-server-status==0.6.0
 django-webpack-loader==0.5.0
 django-oauth-toolkit==1.4.0
 django-user-agents==0.4.0
-djangorestframework==3.9.4
-djoser
+djangorestframework==3.11.2
+djoser==2.0.3
```

Comment:
Can we upgrade to `djoser 2.1.0` since it's the most recent version? Same for `djangorestframework`

Reply:
Upgraded `djangorestframework` to `3.12.4`, `djoser` to `2.1.0` and `social-auth-app-django` to `4.0.0`.